packages/cpp/README.md

```diff
@@ -102,6 +103,38 @@ There are two ways to configure clang-tidy's checks: through a Theia preference
     - for the .clang-tidy file: Checks: "-*,readability-*"
         - Meaning: disable all list-checks and enable all readability-* checks
 
+### Using clang-tidy as a task
+In .theia/tasks.json, add the following:
+```
+   {
+        "label": "[Task] clang-tidy",
+        "type": "shell",
+        "cwd": "${workspaceFolder}",
+        "command": "clang-tidy",
+        "args": [
+            "*"
+        ],
+        "options": {},
+        "problemMatcher": [
+            "$clangTidyMatcher"
+        ]
+   }
+```
+Definition of the fields:
+```
+    "label": <string>            // showed on task selection, after selecting "Run Task..."
```

Comment:
I'm not sure this is an appropriate comment, there are many other instances where `label` is used
besides `Run Task...`. In general a lot of these comments look odd to me since they differ from our own in the code and ultimately what the user will see as help when updating the `tasks.json` themselves.

See: https://github.com/theia-ide/theia/blob/531aa3bde8dea7f022ea41beaee3aace65ce54de/packages/task/src/browser/task-schema-updater.ts#L62

Reply:
Done